File: data_structures_by_c++/arrays/array_input_output.cpp
#include<bits/stdc++.h>
using namespace std;

#define int long long

int32_t main()
{
    // ios::sync_with_stdio(false);
    // cin.tie(nullptr);

    int n;
    cout << "enter number of elements:";
    cin >> n;
    int arr[n];
    cout << "Enter element:" << endl;
    for(int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }
    cout << "Array elements are:";
    for(int i = 0; i < n; i++)
    {
        cout << arr[i] << " ";
    }
}

File: data_structures_by_python/arrays/array_input_output.py
n = int(input("Enter number of elements: "))
arr = []
print("Enter elements:")


# method 1
for i in range(n):
    arr.append(int(input()))

# method 2
# arr = list(map(int, input().split()))
    
# print("Array elements are:")
# for i in arr:
#     print(i)

print("Array elements are:", end=" ")
for x in arr:
    print(x, end=" ")